src/modules/users/user_test.ts

```diff
@@ -300,7 +300,7 @@ describe('test users', () => {
 describe('test users', () => {
   it('update users partner status partner set waiting', async () => {
     await database('users').where('id', userIdPartner).update({
-      status_partner: StatusPartner.WAITING,
```

Comment:
mantaaap, tapi ko aku lebih condong ngasih nama 'UserStatus' ya instead of si 'StatusUser' 😄 
Alasannya karna phrasa si kata sifat (adjective) aja sih, tapi cmiiw wkwkw

Reply:
okey nuuhun kang insight nya 😄 
updated yah [4f76c9a](https://github.com/jabardigitalservice/dedi-core-service/pull/213/commits/4f76c9afe137dcab801b6870d3cd5c38702e832d)